backend/app/api/endpoints/dsh_plugin_storage.py

```
@@ -0,0 +1,115 @@
+)
+from app.services.dsh_plugin_storage import dsh_plugin_storage_service
+
+router = APIRouter()
+
+
+@router.post("/units/{unit}/load", response_model=DshStorageSnapshot)
+def load_unit(
+    unit: str,
+    descriptor: DshStorageDescriptor,
+    package_name: str = Query(alias="package"),
+    current_user: User = Depends(get_current_user),
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)